### ISSUE TITLE
Improvement of chamferCylinder function : specify the number of segments

### ISSUE THREAD

**Issue by gremlinsy** [SebiTimeWaster/Chamfers-for-OpenSCAD#7]
Purpose to have a possibility to define the number of segments.
Useful to create an hexagon (nut) for example.

By mofifying the circleSegments: 
function circleSegments(r, q = -1.0) = ($fn>0) ? $fn : ((r * PI * 4 + 40) * ((q >= 0.0) ? q : globalCircleQuality));

Example to use:
chamferCylinder(r=8,h=8,$fn=8,ch2=0);

**Comment by SebiTimeWaster** [comment]
the commands 7th parameter defines the "circleQuality", setting it low enough creates a hexagon:
```
include <Chamfers-for-OpenSCAD/Chamfer.scad>;

chamferCylinder(10, 10, 10, 1, 1, 360, 0.04);
````

**Comment by gremlinsy** [comment]
thks, it works, but when we change the diameter, it is necessary to recalculate the "circleQuality".
And, it's not really friendly to know the number of segments. 
I think $fn will be more easy.

**Comment by SebiTimeWaster** [comment]
i released a new version v1.2: https://github.com/SebiTimeWaster/Chamfers-for-OpenSCAD/releases/tag/v1.2

a quality parameter of 3 or over will directly put this number throught to the internal $fn parameter instead of calculating the facets internally, take a look into Demo/Demo.scad to see how its used.
